landing-page/content/common/roadmap.md

```diff
@@ -30,33 +30,18 @@ Related design docs will be linked on the planning boards.
 # Priority 1
 
 * API: [Iceberg 1.0.0](https://github.com/apache/iceberg/projects/3) [medium]
-* Spark: [Merge-on-read plans](https://github.com/apache/iceberg/projects/11) [large]
-* Maintenance: [Delete file compaction](https://github.com/apache/iceberg/projects/10) [medium]
-* Flink: [Upgrade to 1.13.2](https://github.com/apache/iceberg/projects/12) (document compatibility) [medium]
 * Python: [Pythonic refactor](https://github.com/apache/iceberg/projects/7) [medium]
+* Spec: [Z-ordering / Space-filling curves](https://github.com/apache/iceberg/projects/16) [medium]
+* Spec: [Snapshot tagging and branching](https://github.com/apache/iceberg/projects/4) [small]
+* Views: [Spec](https://github.com/apache/iceberg/projects/6) [medium]
 
 # Priority 2
 
 * ORC: [Support delete files stored as ORC](https://github.com/apache/iceberg/projects/13) [small]
 * Spark: [DSv2 streaming improvements](https://github.com/apache/iceberg/projects/2) [small]
 * Flink: [Inline file compaction](https://github.com/apache/iceberg/projects/14) [small]
 * Flink: [Support UPSERT](https://github.com/apache/iceberg/projects/15) [small]
-* Flink: [FLIP-27 based Iceberg source](https://github.com/apache/iceberg/projects/23) [large]
```

Comment:
Do you know if this is finished entirely I'd just clozs enough to remove from the roadmap?

Reply:
I think so--all of the PRs in the project are merged: https://github.com/apache/iceberg/projects/23

Although this was opened 2 days ago: https://github.com/apache/iceberg/pull/5191

Reply:
@samredai FLIP-27 is not completely done yet. MVP version is merged. we still have some follow-up items.

Reply:
Thanks @stevenzwu! I added it back and moved it to Priority 1.